### ISSUE TITLE
Screen corruption with output=ttf, ttf.cols=90, and fastbioslogo=true

### ISSUE THREAD

**Issue by emendelson** [joncampbell123/dosbox-x#2114]
Tested under SDL1 and SDL2 in Windows and SDL1 on the Mac:

With a config file that contains only this:

```
[sdl]
output  = ttf

[render]
ttf.cols = 90

[dosbox]
fastbioslogo = true
```

the screen is corrupted. The lefthand ten characters are repeated at the right. This does not occur with fastbioslogo=false.

**Comment by Wengier** [comment]
Thanks for reporting. I could confirm that the problem occurred with "fastbioslogo=true" (or with command-line option -fastlaunch), and I believe I have already fixed it in #2115.

**Comment by emendelson** [comment]
@Wengier Yes. Fixed. Thank you!
